test/e2e/operator_test.go

```diff
@@ -228,3 +252,195 @@ func setImage(deployment *appsv1.Deployment, image string) {
 		}
 	}
 }
+
+func TestDNSForwarding(t *testing.T) {
+	cl, err := getClient()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// Create the upstream resolver ConfigMap.
+	upstreamPodName := "test-upstream"
+	upstreamPodNS := "openshift-dns"
+	upstreamCfgMap := upstreamConfigMap(upstreamPodName, upstreamPodNS)
```

Comment:
Now that `upstreamConfigMap` and `upstreamCorefile` are in separate files, it might make sense to pass `upstreamCorefile` in as an argument.

Reply:
ack